src/lib/components/Carousel/imageCarousel.svelte

```diff
@@ -0,0 +1,62 @@
+<script lang="ts">
```

Comment:
Made a dynamic image carousel - parameters required are:
* `images` - an array of imported images - see https://github.com/lhollyer/the-small-society/pull/17/files#diff-864cd7499125efdabc67a4236aa1d65f51aa4848c8cc748e51b0a497aea550caR17 for example implementation
* `carouselId` - unique HTML ID for each carousel - see https://github.com/lhollyer/the-small-society/pull/17/files#diff-864cd7499125efdabc67a4236aa1d65f51aa4848c8cc748e51b0a497aea550caR62 on how to pass through these props/parameters